### PR TITLE
updated navbar with defaultlogo for anonymous users

### PR DESCRIPTION
**Pull request by abilpraju-aot**
https://aottech.atlassian.net/browse/FWF-2293
image
image

### CODE REVIEW

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-nav/src/Navbar.jsx [hunk 1]
```diff
@@ -139,6 +141,14 @@ const NavBar = React.memo(({ props }) => {
     }
   }, [instance]);
 
+  React.useEffect(() => {
+    if (location.pathname.includes('public') && !isAuthenticated) {
+      setApplicationTitle("formsflow.ai");
```
Comment (abhilash-aot) [inline forms-flow-nav/src/Navbar.jsx:146]: Don’t we have a generic logo change page ?

Comment (abhilash-aot) [reply forms-flow-nav/src/Navbar.jsx:146]: Also application title comes from env for cases other than multitenancy

**Review by abhilash-aot — CHANGES_REQUESTED** [review]
Check for the generic change adding logo and default the values instead to take these values and logo rather than hardcoding

File: forms-flow-nav/src/Navbar.jsx (hunk 1 above)
Comment (shuhaib-aot) [reply forms-flow-nav/src/Navbar.jsx:146]: I think if the form name is public this useEffect will work. please check that

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

**Pull request closed without merging**